### ISSUE TITLE
Android availability

### ISSUE THREAD

**Issue by ohhai** [Loirooriol/tab-counter-plus#25]
This extension seems working fine on Android, but addons.mozilla.org site tells that it's not for Android.
I had to point FF to the .xpi file directly.
Maybe just manifest file update needed?

Side request, might be worth of separate ticket: it would be great to see the (overall) tabs number in main menu of FF for Android. Currently it requires an extra click to see counters (and the new page affects counter by itself?).

**Comment by ohhai** [comment]
![Screenshot_2020-06-17-23-34-49](https://user-images.githubusercontent.com/3885783/84947827-90bbf500-b0f3-11ea-8162-a7ea155b64a6.png)

My screenshot with Ublock counter as example.

**Comment by Loirooriol** [comment]
I think in some case the add-on can use some API which was not available on Android, I don't remember much. Anyways I didn't really test it on Android since Firefox for Android has a native tab counter, which seemed to work fine. I didn't know it can only reach 99, I only have thousands of tabs on desktop.

Showing the number in the menu seems an improvement indeed.

**Comment by ohhai** [comment]
If that is some desktop API - it should be something not related to main tab counting, since it works fine. I can try to check it, if FF for Android has some way to (enable remote debugging and search for errors?:)

**Comment by Loirooriol** [comment]
OK, so the `browser.windows` API doesn't work. I guess that's why you don't see the number of tabs in the menu, that's probably the title or the badge, but the background script is not working correctly. `browser.windows` is also used in the windowId polyfill, but I should just remove it since it has been quite some time since I implemented windowId in Firefox.

**Comment by ohhai** [comment]
Sounds reasonable, I'm not sure if it's possible to have windows number > 1 for Android (excluding private mode maybe).

**Comment by Loirooriol** [comment]
No, android has a single internal window. It's just the UI that makes it look like there are two of them. So to be consistent, I guess the add-on would need to query the tabs with the same `incognito` state.
